src/sentry/tasks/derive_code_mappings.py

```diff
@@ -71,6 +73,12 @@ def derive_code_mappings(
     try:
         with lock.acquire():
             trees = installation.get_trees_for_org()
+    except ApiError as error:
```

Comment:
When we call get_trees_for_org, we need to request access tokens from the GH installation for that org.
If the admins, uninstalled the Sentry GH App we would then not be able to request tokens anymore.
If you load [the Sentry issue you](https://sentry.io/organizations/sentry/issues/3788707673/?project=1&query=is%3Aunresolved+assigned%3A%5Bme%2C+none%5D+timesSeen%3A%3E10+derive&referrer=issue-stream&sort=freq&statsPeriod=2h) will see that `HTTPError 404 Client Error: Not Found for url: https://api.github.com/app/installations/<some_id>/access_tokens` is raised at the bottom of the stack trace.